platform_plugin_superset/superset_xblock.py

```diff
@@ -75,7 +75,7 @@ class SupersetXBlock(XBlock):
 
     filters = List(
         display_name=_("Filters"),
-        help=_("""Semicolon separated list of filters to use. E.g: org="edX"; country in ("us", "co").
+        help=_("""Semicolon separated list of filters to use. E.g: org='edX'; country in ('us', 'co').
```

Comment:
Question: what syntax do these filters use?

Reply:
can we specify it in the description? Thanks!

Reply:
updated